### ISSUE TITLE
Pokemon Classifier

### ISSUE THREAD

**Issue by AgastyaPatel** [AgastyaPatel/FastAI-course#2]
Pokemon Classifier:
Working on pokemon classifier along with making some comparisons between models, effect of learning rate and epoch.

Model Deployed: [Hugging Face Spaces](https://huggingface.co/spaces/AgastyaPatel/Pokemon-Classifier)

**Comment by AgastyaPatel** [comment]
Commit: eb1a75ed8629289bca79f7fd4d5b3e893e8b55c5
- Resnet 18 model
- Resnet 18 with good learning rate
- Comparing training performance of the models also with having different epochs
Issue: Model has high accuracy with downloaded dataset and seems to underperform with real world use case

Commit: f008400f5b502c0d6f10c13c1b4de8054adacf59
- Added Resnet34, Resnet34 with good learning rate
- Fixed accuracy of Resnet18 models with data-augmentation

**Comment by AgastyaPatel** [comment]
Closing the project as trying multiple pretrained models like resnet18, resnet34, resnet50 and convnext_nano have all resulted in similar accuracy.
